Task2/dashboard.js

```diff
@@ -0,0 +1,246 @@
+// fetch('https://reqres.in/api/users')
+// .then(res=>res.json())
+// .then(data=>console.log(data))
+
+// let email=localStorage.getItem('email')?localStorage.getItem('email'):''
+// console.log(email);
+// if(email=='')
+// {
+//     alert('u need to login first');
+//     window.location.href="index.html";
+// }
+
+const root = document.getElementById("root");
+const form = document.getElementById("Form");
+const Post = document.getElementById("btn");
+// const list=document.createDocumentFragment();
+const url = "https://reqres.in/api/users";
+let users = new Array();
+console.log(users);
+async function Get() {
+  try {
+    const res = await fetch(url);
+    //console.log(res);
+    const data = await res.json();
+    //console.log(data);
+    let persons = data;
+    //console.log(data);
+    // console.log(persons.data)
+    persons.data.map((values) => {
```

Comment:
Why does the need to create a new variable and pass data to it? Also, again need to destructure properties here (and the section below as well).

```
const dataResponse = await fetch(URL);
const personsData = await dataResponse.json();

personsData.map(({ first_name, email, avatar } => { ... })
```

@Poorvi-3pillar